pkgs/cupertino_http/lib/src/cupertino_client.dart

```diff
@@ -279,6 +279,15 @@ class CupertinoClient extends BaseClient {
       throw ClientException('Redirect limit exceeded', request.url);
     }
 
+    final headers = response.allHeaderFields
+        .map((key, value) => MapEntry(key.toLowerCase(), value));
```

Comment:
The Dart SDK has traditionally changed headers to lower case and we ended up needing to add an option to _not_ do that, because some servers _do_ care about header case.

See https://github.com/dart-lang/http/issues/609

Reply:
These are response headers. This is just moving some code so that the `'content-length'` header can be examined